Gems/ScriptCanvas/Assets/TranslationAssets/Classes/MaterialData.names

```diff
@@ -11,7 +11,6 @@
             "methods": [
                 {
                     "base": "GetBaseColor",
-                    "context": "MaterialData",
```

Comment:
Why do we need to remove this context field only? and as I see you remove
```
methodEntry.m_context = className;
```
seems like we don't need context field for method level?